pulp_ansible/app/tasks/utils.py

```diff
@@ -216,3 +218,16 @@ def get_file_obj_from_tarball(tar, file_path, artifact_path, raise_exc=True):
         raise FileNotFoundError(f"{file_path} not found")
 
     return file_obj
+
+
+def get_last_sync_task(pk):
+    task = (
+        Task.objects.filter(reserved_resources_record__icontains=pk, name__icontains="sync")
```

Comment:
```suggestion
        Task.objects.filter(reserved_resources_record__contains=pk, name__contains="sync")
```
We don not need to search case insensitive.